tests/testhttp.nim

```diff
@@ -163,33 +31,11 @@ httpsrv.rpc("noParamsProc") do():
 
 httpsrv.start()
 
-suite "HTTP Server/HTTP Client RPC test suite":
+suite "JSON-RPC test suite":
+  test "Simple RPC call":
+    check waitFor(simpleTest("localhost", Port(8545))) == true
   test "Continuous RPC calls (" & $TestsCount & " messages)":
     check waitFor(continuousTest("localhost", Port(8545))) == TestsCount
-  test "Wrong [Content-Type] test":
```

Comment:
we're not testing HTTP Server + Client anymore, we just assume that's done in `nim-chronos`